pkg/debugapi/tag_test.go

```diff
@@ -260,8 +247,8 @@ func TestTags(t *testing.T) {
 		if finalTag.Total != 3 {
 			t.Errorf("tag total count mismatch. got %d want %d", finalTag.Total, 3)
 		}
-		if finalTag.Seen != 3 {
-			t.Errorf("tag seen count mismatch. got %d want %d", finalTag.Seen, 3)
+		if finalTag.Seen != 1 {
```

Comment:
this has changed due to a bug in the mock storage that resulted in incorrect count of `seen` chunks on insertion